promptsource/templates/swag/regular/templates.yaml

```diff
@@ -2,240 +2,130 @@ dataset: swag
-      original_task: null
+      metrics:
+      - Accuracy
+      - ROUGE
+      original_task: true
     name: first_then
-    reference: ''
-  68d88e46-3a74-4059-927d-944e4b37ad75: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 68d88e46-3a74-4059-927d-944e4b37ad75
-    jinja: "Consider the sentence: {{ sent2 }} {{ [ending0, ending1, ending2, ending3][2]\
-      \ }} \nIs it an appropriate continuation of the following situation:\n{{ sent1\
-      \ }} ?\nYes or No?\n||| {{ {0: \"No\", 1: \"No\", 2: \"Yes\", 3: \"No\"}[label]\
-      \ }}"
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_reversed_2
-    reference: ''
-  875dacc7-18b6-4b21-a6fb-8e8afabf86a7: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 875dacc7-18b6-4b21-a6fb-8e8afabf86a7
-    jinja: 'Consider the sentence: {{ sent1 }}
-
-      Is the following an appropriate continuation?
-
-      {{ sent2 }} {{ [ending0, ending1, ending2, ending3][3] }}
-
-      Yes or No?
-
-      ||| {{ {0: "No", 1: "No", 2: "No", 3: "Yes"}[label] }}'
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_3
-    reference: ''
-  96adb760-2714-4feb-b174-b1ec03622c89: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 96adb760-2714-4feb-b174-b1ec03622c89
-    jinja: "If a description of a situation begins like this: {{ startphrase }}...\
-      \ Then how does it continue? \nEnding 1: {{ ending0 }}\nEnding 2: {{ ending1\
-      \ }}\nEnding 3: {{ ending2 }}\nEnding 4: {{ ending3 }}\n\n||| {{ {0: \"Ending\
-      \ 1\", 1: \"Ending 2\", 2: \"Ending 3\", 3: \"Ending 4\"}[label] }}"
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: if_begins_how_continues
-    reference: ''
-  a453f563-16d0-4021-bc00-a3806e70c589: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: a453f563-16d0-4021-bc00-a3806e70c589
-    jinja: 'Consider the sentence: {{ sent1 }}
-
-      Is the following an appropriate continuation?
-
-      {{ sent2 }} {{ [ending0, ending1, ending2, ending3][1] }}
-
-      Yes or No?
-
-      ||| {{ {0: "No", 1: "Yes", 2: "No", 3: "No"}[label] }}'
+    reference: Generate the ending
+  d61be86a-242e-48ad-871c-f8be5133c4df: !Template
+    answer_choices:
+    - (a)
+    - (b)
+    - (c)
+    - (d)
+    answer_choices_key: ending0 ||| ending1 ||| ending2 ||| ending3
+    id: d61be86a-242e-48ad-871c-f8be5133c4df
+    jinja: "First, {{ sent1.lower() }} Then, {{ sent2.lower() }}... \nChoose the key\
+      \ with an appropriate ending:\n(a) {{ ending0 }}\n(b) {{ ending1 }}\n(c) {{\
+      \ ending2 }}\n(d) {{ ending3 }}\n||| \n{{answer_choices[label]}}\n"
```

Comment:
This one doesn't follow the correct format. If indices like `(a)` are given, then those should be used as the answers. You also shouldn't set both answer_choices and answer_choices_key.